src/unk_0203BBB4.c

```diff
@@ -0,0 +1,36 @@
+#include "unk_0203BBB4.h"
+
+const struct MonthDay DiamondDustDates[] = {
```

Comment:
This can be static, since no other files use it. (and since it's static, the name should have an `s` prefix)
```suggestion
static const struct MonthDay sDiamondDustDates[] = {
```

Reply:
Making this static caused there to be a hash error, so I'll leave it as a const for now since it works just fine